plugins/aws_workspaces/icon_aws_workspaces/actions/create_workspace/schema.py

```diff
@@ -86,50 +86,10 @@ class CreateWorkspaceInput(komand.Input):
     "username"
   ],
   "definitions": {
-    "tag": {
```

Comment:
should we have lost this input type in the schema? looks like we lost the custom type being specified, as the action still takes it but is now a string